Reading List/Reading List/Model Controllers/BookController.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  BookController.swift
+//  Reading List
+//
+//  Created by Nathanael Youngren on 1/17/19.
+//  Copyright © 2019 Lambda School. All rights reserved.
+//
+
+import Foundation
+
+class BookController {
+    
+    init() {
+        loadFromPersistence()
+    }
+    
+    func createBook(title: String, reasonToRead: String) {
+        let newBook = Book(title: title, reasonToRead: reasonToRead)
+        books.append(newBook)
+        saveToPersistence()
+    }
+    
+    func updateHasBeenRead(for book: Book) {
+        guard let index = books.index(of: book) else { return }
+        books[index].hasBeenRead = !books[index].hasBeenRead
```

Comment:
This is new Swift 4.2 feature, which toggles `Bool` value for you
```suggestion
        books[index].hasBeenRead.toggle()
```